cmd/earthly/root_cmds.go

```diff
@@ -287,6 +287,13 @@ Set up a whole custom git repository for a server called example.com, using a si
 			},
 			Subcommands: app.secretCmds(),
 		},
+		{
+			Name:        "registry",
+			Aliases:     []string{"registries"},
+			Description: "Manage registry access *experimental*",
+			Usage:       "Manage registry access *experimental*",
```

Comment:
I *think* we have to make this *beta* from the get-go, given that it's critical for CI.